src/pages/Error.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="flex flex-col space-y-8 items-center">
+    <!-- 404 -->
+    <div v-if="isError404" class="w-full mt-16 flex flex-col space-y-4 items-center">
+      <inline-svg class="w-12 h-12 text-yellow-600 fill-current" :src="warningSvg" />
+
+      <p class="text-2xl align-middle">Page Not Found</p>
+      <p class="text-lg text-gray-500 text-center w-10/12 sm:w-1/2">
+        We are unable to find what you are looking for
```

Comment:
This page comes when required params are not passed, no? Maybe have a more descriptive error?